packages/flutter/lib/src/material/radio_list_tile.dart

```diff
@@ -488,11 +488,15 @@ class RadioListTile<T> extends StatelessWidget {
         );
     }
 
+    final ListTileThemeData listTileTheme = ListTileTheme.of(context);
+    final ListTileControlAffinity effectiveControlAffinity =
+        this.controlAffinity ?? listTileTheme.controlAffinity ?? ListTileControlAffinity.platform;
```

Comment:
```suggestion
        controlAffinity ?? listTileTheme.controlAffinity ?? ListTileControlAffinity.platform;
```

We should avoid using `this`

Reply:
updated